gravitee-gateway-services/gravitee-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/apikeys/task/ApiKeyRefresher.java

```diff
@@ -137,17 +138,19 @@ public void run() {
     }
 
     private void saveOrUpdate(ApiKey apiKey) {
+        String cacheKey = ApiKeysCacheService.buildCacheKey(apiKey);
+
         if (apiKey.isRevoked() || apiKey.isPaused()) {
             logger.debug(
-                "Remove a paused / revoked api-key from cache [key: {}] [plan: {}] [app: {}]",
-                apiKey.getKey(),
+                "Remove a paused / revoked api-key from cache [id: {}] [plan: {}] [app: {}]",
```

Comment:
don't know if we should log `apiKey.id` or `cacheKey`

Reply:
I intentionally never logged API keys, cause I think it's a bad security practice IMO, it's like logging a password.
Do you agree with that ?